src/main/kotlin/help/HelpWindow.kt

```diff
@@ -37,13 +37,17 @@ class HelpWindow(
     override fun show(state: HelpViewState) {
         content.removeAllComponents()
         when(state) {
-            is LoadingHelpViewState -> onLoading(state)
+            is LoadingHelpViewState -> onLoading()
             is LoadedHelpViewState -> onLoaded(state)
         }
     }
 
-    private fun onLoading(state: LoadingHelpViewState) {
-        println("onLoading:: $state")
+    private fun onLoading() {
+        listOf(LineBreak(),PlainText("Loading...."),LineBreak()).map { it.toComponent()}
```

Comment:
This could be formatted better, with spaces between arguments and each operator on a newline:
```kotlin
listOf(LineBreak() ,PlainText("Loading...."), LineBreak())
  .map { it.toComponent()}
  .map { content.addComponent(it) }
  .forEach {
    it.setLayoutData(LinearLayout.createLayoutData(Fill))
  }
```